tsconfig.json

```diff
@@ -16,5 +16,5 @@
     "skipLibCheck": true
   },
   "include": ["**/*.ts", "**/*.d.ts"],
-  "exclude": ["node_modules", "**/coverage/*", "test"]
+  "exclude": ["node_modules", "**/coverage/*", "test", "dist"]
```

Comment:
Did you find that this was needed? I'm assuming Typescript is smart enough to exclude the "outDir" folder ("dist")

Reply:
Could be that the TS compiler is smart enough, but I think the default could probably be overwritten by the include of `"**/*.d.ts"`. With `rimraf` it would not be necessary anymore, but you still are on the safer side with it and it does not hurt either.